man/systemd.exec.xml

```diff
@@ -1602,6 +1602,38 @@
         functions) if their standard output or standard error output is connected to the journal anyway, thus enabling
         delivery of structured metadata along with logged messages.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>$SERVICE_RESULT</varname></term>
+
+        <listitem><para>Only defined for the service unit type, this environment variable is passed to all
+        <varname>ExecStop=</varname> and <varname>ExecStopPost=</varname> processes, and encodes the service
+        "result". Currently, the following values are defined: <literal>timeout</literal> (in case of an operation
+        timeout), <literal>exit-code</literal> (if a service process exited with a non-zero exit code; see
+        <varname>$EXIT_STATUS</varname> below for the actual exit status returned), <literal>signal</literal> (if a
+        service process was terminated abnormally by a signal; see <varname>$EXIT_STATUS</varname> below for the actual
+        signal used for the termination), <literal>core-dump</literal> (if a service process terminated abnormally and
+        dumped core), <literal>watchdog</literal> (if the watchdog keep-alive ping was enabled for the service but it
```

Comment:
> <literal>start-limit-hit</literal> (the maximimum number of starts within the configured time was reached)

We will never see `SERVICE_RESULT=start-limit-hit`
```sh
-bash-4.3# systemctl cat start-limit-hit --no-pager
# /etc/systemd/system/start-limit-hit.service
[Service]
ExecStart=/bin/sh -x -c 'exit 1'
Restart=always
ExecStopPost=/bin/sh -x -c 'set'

-bash-4.3# systemctl start-limit-hit

-bash-4.3# systemctl show --property Result start-limit-hit --no-pager
Result=start-limit-hit

-bash-4.3# journalctl -u start-limit-hit -o cat | grep SERVICE_RESULT
SERVICE_RESULT=exit-code
SERVICE_RESULT=exit-code
SERVICE_RESULT=exit-code
SERVICE_RESULT=exit-code
SERVICE_RESULT=exit-code
```
Shouldn't we remove this from the manpage. The same is for `timeout`
```sh
-bash-4.3# systemctl cat timeout --no-pager
# /etc/systemd/system/timeout.service
[Service]
Type=notify
TimeoutSec=5s
ExecStart=/bin/bash -x -c 'sleep 20'
ExecStop=/bin/bash -x -c 'set'
ExecStopPost=/bin/bash -x -c 'set'

-bash-4.3# systemctl start timeout

-bash-4.3# journalctl -u timeout -o cat | grep SERVICE_RESULT

-bash-4.3# systemctl show --property Result timeout --no-pager
Result=timeout
```